rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidPatientRegisterActivityTest.java

```diff
@@ -35,7 +35,6 @@ public void setUp() {
         MockitoAnnotations.initMocks(this);
         covidPatientRegisterActivity = Robolectric.buildActivity(CovidPatientRegisterActivity.class)
                 .create()
-                .resume()
```

Comment:
Why is this removed?

Reply:
basically, the `userVerification` method is calling in the onResume method and Robolectric failed on `syncUtils.verifyAuthorization()` method

Reply:
Test coverage is failing because of that. Mock the method call if that causes a failure.

Reply:
done